src/domain.py

```diff
@@ -469,6 +466,11 @@ def domain_remove(
             domain=domain, recovery_password=dyndns_recovery_password
         )
 
+    rm(f"/etc/yunohost/certs/{domain}", force=True, recursive=True)
+    for key_file in glob.glob(f"/etc/yunohost/dyndns/K{domain}.+*"):
+        rm(key_file, force=True)
+    rm(f"{DOMAIN_SETTINGS_DIR}/{domain}.yml", force=True)
```

Comment:
Hmmm I'm skeptical about moving this after the regenconf ... or was the intent to run the `dyndns_unsubscribe` before the dyndns key is deleted ?